src/coreclr/src/vm/method.cpp

```diff
@@ -4865,6 +4865,16 @@ bool MethodDesc::IsJitOptimizationDisabled()
 {
     WRAPPER_NO_CONTRACT;
 
+#ifdef FEATURE_INTERPRETER
+    static ConfigDWORD s_ForceInterpreterTier0;
+    static ConfigDWORD s_ForceInterpreterAlways;
+    if (s_ForceInterpreterAlways.val(CLRConfig::INTERNAL_ForceInterpreterAlways) == 0 &&
```

Comment:
The check for ForceInterpreterAlways appears unneeded here? If ForceInterpreter==1 then the method is going to be interpretted regardless of whether jit optimization is allowed.

Reply:
The check for `ForceInterpreterAlways` _is_ unneeded here. I just added this check to minimize the change from the original codebase. I can remove this check.

Reply:
Done in commit https://github.com/dotnet/runtime/pull/37825/commits/3345ef30ec343156746d028112c34a78cd67851a.